setenv.sh

```diff
@@ -1,14 +1,31 @@
-# if running the kdb+tick example, change these to full paths
-# some of the kdb+tick processes will change directory, and these will no longer be valid
-export TORQHOME=${PWD}
-export KDBCONFIG=${TORQHOME}/config
-export KDBCODE=${TORQHOME}/code
-export KDBLOG=${TORQHOME}/logs
+export TORQHOME=${PWD}                                                                              # if running the kdb+tick example, change these to full paths
+export TORQDATA=${PWD}                                                                              # some of the kdb+tick processes will change directory, and these will no longer be valid
+export TORQAPPHOME=${PWD}
+export KDBLOG=${TORQDATA}/logs
 export KDBHTML=${TORQHOME}/html
 export KDBLIB=${TORQHOME}/lib
+export KDBAPPCODE=${TORQAPPHOME}/code
+export KDBCONFIG=${TORQAPPHOME}/config
```

Comment:
This should be TORQHOME

And we should also have 

KDBAPPCONFIG=${TORQAPPHOME}/config